todo/src/reducers/index.js

```diff
@@ -0,0 +1,60 @@
+import {ADD_TODO, TOGGLE_COMPLETED, DELETE_ITEM} from '../actions';
+
+let initialState;
+if (localStorage.getItem('state')) {
```

Comment:
Great job on getting local storage working. I hadn't seen anybody do it in my old cohort, I've seen like 4 of you guys do it today. You guys are on fire.